repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -570,7 +570,9 @@ export const getPreassignedFareProducts = (
             version: '1.0',
             id: `op:Pass@${product.productName}_${passengerType}`,
             Name: {
-                $t: `${product.productName} Pass - ${passengerType}`,
+                $t: isFlatFareType(userPeriodTicket)
+                    ? product.productName
+                    : `${product.productName} - ${passengerType}`,
```

Comment:
change this ${product.productName} Pass - ${passengerType}

Reply:
I think the requirements have changed